src/style.css

```diff
@@ -1 +1,76 @@
 /* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;700&display=swap");
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+.header {
+  min-width: 1024px;
+  max-width: 1200px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  height: 60px;
+  align-items: center;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+  padding-right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  list-style: none;
+}
+
+.nav__item {
+  list-style-type: none;
+  text-transform: uppercase;
+  margin-left: 20px;
+}
+
+.is-active {
+  color: #00acdc;
+  text-decoration: none;
+  font-weight: 500px;
+}
+
+.is-active::after {
+  position: relative;
+  top: 18.5px;
```

Comment:
Don't use decimal px values. Also, don't hardcode such distances, because it will break if header height changes.
Try to find different reference points, for example, bottom side of the element.